src/app/components/cohort-selection/cohort-selection.directive.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Created by bo on 9/11/16.
+ */
+'use strict';
+
+angular.module('transmartBaseUi')
+    /**
+     * Represents the cohort-selection section inside the workspace,
+     * the user can create multiple cohort-selection sections.
+     * @memberof transmartBaseUi
+     * @ngdoc directive
+     * @name cohortSelection
+     */
+    .directive('cohortSelection', function () {
+        return {
+            restrict: 'E',
+            templateUrl: 'app/components/cohort-selection/cohort-selection.tpl.html',
+            controller: 'CohortSelectionCtrl as cohortSelectionCtrl',
```

Comment:
`controller: 'CohortSelectionCtrl as cohortSelectionCtrl',`
this is unnecessary